tmt/steps/report/reportportal.py

```diff
@@ -49,6 +51,11 @@ class ReportReportPortalData(tmt.steps.report.ReportStepData):
         metavar="NAME",
         default=None,
         help="The launch name (base name of run id used by default).")
+    metadata: Optional[str] = tmt.utils.field(
+        option="--metadata",
+        metavar="METADATA",
```

Comment:
How about changing this to `"JSON"`? Then we would get `--metadata JSON` which would be more helpful for me as a user than `--metadata METADATA` as it describes the expected format.

Reply:
Ack